### ISSUE TITLE
Include images stored in subfolders 

### ISSUE THREAD

**Issue by ueberf** [Niels-IO/next-image-export-optimizer#6]
First of all, thank you for writing this plugin and finally bringing a solution to the next.js image export shenanigan! ❤️

From the docs and tryouts I understand that images have to be stored in a folder like `public/images`. 
It would be super nice if subfolders would get included in the optimization process, too, so that 

```
next-app/
├─ public/
│  ├─ images/
│  │  ├─ hero-image.jpg
│  │  ├─ project-1/
│  │  │  ├─ project-cover.jpg
│  │  ├─ project-2/
│  │  │  ├─ project-cover.jpg/
│  │  ├─ .../

```

gets compiled into

```
next-app/
├─ out/
│  ├─ images/
│  │  ├─ nextImageExportOptimizer/
│  │  │  ├─ hero-image-32.webp
│  │  │  ├─ hero-image-16.webp
│  │  │  ├─ ...
│  │  │  ├─ project-1/
│  │  │  │  ├─ project-cover-16.webp
│  │  │  │  ├─ project-cover-32.webp
│  │  │  │  ├─ ...
│  │  │  ├─ project-2/
│  │  │  │  ├─ project-cover-16.webp/
│  │  │  │  ├─ project-cover-32.webp
│  │  │  │  ├─ ...
│  │  │  ├─ .../

```

Like this, the public image folder would stay nice and organised for bigger projects without having to worry about conflicting image names.

**Comment by Niels-IO** [comment]
Hi @ueberf, 
I just published version 0.7.1 which enables the use of subfolders. Could you please give it a try with your setup?

**Comment by ueberf** [comment]
Hey @Niels-IO,

Just had the chance to try out version 0.7.1, and it works like a charm!! 

Thanks a bunch for the quick update!
